### ISSUE TITLE
Questions on DeLaN Reproduction: High Training Loss & Poor Torque Prediction

### ISSUE THREAD

**Issue by dzg2szh** [milutter/deep_lagrangian_networks#7]
Dear Authors,
    I am attempting to reproduce your code on my own robot dataset. Here are the relevant details:
    **Robot**: A 6-degree-of-freedom (DoF) fairion FR5 robotic arm manufactured in China.
    **Dataset/Trajectories:** The link below contains the data recorded during the robot's operation. The trajectories consist of concatenated point-to-point motions and circular motions.
At that time, GitHub Issues does not support uploading files in pickle format, so the pickle file has been separately sent to your email address (Lutter@ias.tu-darmstadt.de) via email.
    **Specifications:** The data was collected at a frequency of 250Hz and converted to the pickle format required by the DeLaN network. The pickle file only includes joint angles, joint angular velocities, joint angular accelerations, and joint torques.

    **However, I encountered two issues:**
    1.The training loss stabilizes at around 4300 epochs with a value of 5.8e+00, which is significantly larger than the loss reported in your work.

<img width="1010" height="860" alt="Image" src="https://github.com/user-attachments/assets/d610575a-14d2-46df-8e3c-198af1fd471a" />

    2.When testing on the test set, the predicted joint torques exhibit very poor performance.
         <1> the original torque in dataset
                   
<img width="541" height="402" alt="Image" src="https://github.com/user-attachments/assets/f128535d-7eac-43f1-adf7-e34a2c2bdb52" />
         <2> torque comparsion between original torque and prediction for 6 joints

<img width="835" height="680" alt="Image" src="https://github.com/user-attachments/assets/706a3108-627f-477e-8a3b-a8a6f0c2e884" />

    Could you please provide some guidance on these problems? Thank you very much for your help!
